container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -755,6 +755,9 @@ roles:
   - name: blobstore_url_signer
   - name: route_registrar
   - name: metron_agent
+  tags:
+  - indexed
+  # Role needs LB => indexed
```

Comment:
The comment should be describing _why_ it needs an LB.

Reply:
Added info in commit 5ece0bad56d95b4351fcf8c95cf559af9d89fcfa